snappy_wrappers/wrappers/bcftools/TMB/wrapper.py

```diff
@@ -32,24 +32,50 @@
 number_snvs=$(bcftools view -R $bed_file -v snps --threads 2 -H {snakemake.input.vcf}| wc -l)
 number_indels=$(bcftools view -R $bed_file -v indels --threads 2 -H {snakemake.input.vcf}| wc -l)
 number_variants=$(bcftools view -R $bed_file --threads 2 -H {snakemake.input.vcf}| wc -l)
+number_missense_variants=$(bcftools view -R $bed_file --threads 2 -H {snakemake.input.vcf}| grep 'missense_variant'| wc -l)
```

Comment:
Please make sure that it is safe with any annotation software (at least with vep & jannovar).